### ISSUE TITLE
[Broken Website] fritz.box

### ISSUE THREAD

**Issue by IzzySoft** [darkreader/darkreader#12115]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [ ] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Starting only recently (worked all the time before), https access is no longer possible with DarkReader enabled. Connection is stuck forever at "Establishing a TLS-Handshake to fritz.box…". As soon as I disable DarkReader, the site loads.

DarkReader has only minimal CSS configured for that site (as seen in its Dev Tools) which I doubt cause that, so I'm not sure what exactly is causing this.

### Website Address

192.168.178.1

### Steps To Reproduce

Just call up `https://fritz.box/` and see … nothing (except the status messages stating connection progress). There is some data transferred, then the screen turns dark (which is where I guess DarkReader kicks in), then the connection stalls as described.

I've disabled uBO in case it might be related, no avail. If I disable DarkReader, I get through – and once logged in, I can re-enable it and it seems working (so it seems it's only blocking the login; I've tried enabling it once the login page is shown, but then it hangs sending the login information).

I've tried in a fresh profile with DarkReader being the only addon (freshly installed) and funnily could not reproduce the issue there. But I always can reproduce it with my main profile. I've checked the other addons I'm using, but apart from uBlock Origin and uMatrix (which I both disabled for testing, without avail) I see no other that could be related.

### Operating System

Linux

### Web Browser name and version

Waterfox-G 6.0.7

### Dark Reader version

4.9.66

### Additional Context

None of all the other sites I use seem to be affected. Though I saw a comparable issue at work (MSEdge on Windows-10 when calling orangesharing.com (running Confluence), which takes ages to load as seemingly DarkReader generates a huge stylesheet that tries to load graphics from several Confluence servers, some of them with `.ru` domains. No page finished loading faster than 10..15 min, some took up to an hour. That is reproducible in a fresh profile.

I didn't update DarkReader beyond 4.9.66 (or rather had go back to this one) for some other issues I had when updating to 4.9.70 or 71; something seemed to have it broken entirely then (I don't remember details, but it seemed to no longer work at all).

Apologies for ticking the "can reproduce on clean profile", but without that I cannot send the report. Still hoping for a clue what might cause the problem and how to fix it, even if I have to fix it on my end.

**Comment by Myshor** [comment]
As I understand you have problem to connect to your router when Dark Reader is enabled.

Hard to do any investigation or fixes when we don't use that kind of routers.

And you write: "I didn't update DarkReader beyond 4.9.66".
"I've tried in a fresh profile with DarkReader being the only addon (freshly installed) and funnily could not reproduce the issue there. "
So it seems it's working fine with latest Dark Reader.

This gives me one obvious conclusion - we won't fix it, as it's working already fine with newest version.

**Comment by IzzySoft** [comment]
> So it seems it's working fine with latest Dark Reader.

I doubt that a little (as outlined, I have a comparable issue at work with a different site, and there's the latest DarkReader running). I was hoping for a clue what other addon might be colliding with DarkReader.

Well, for now my work-around is: Disabling DarkReader for fritz.box, logging in to the box, enabling it again. That works fine so far (so it seems just the initial loading and login process being blocked for some reason), and luckily I don't need to check things on the box too often…

I'll certainly check with a newer release one day (guess the issues I had with the last update were caused around 4.9.69 and fixed around 4.9.70/71, but I'm not sure; IIRC I was upgrading to 4.9.70 back then. Was a little hazzle to downgrade back again, which kept me from trying newer versions until now.

Does it make sense to reopen this issue should the problems remain after an upgrade, though it seems to work in a fresh profile? I understand you're saying it works there – but should it be a conflict between multiple addons, where to report it if I don't know which the other one might be?

Similar question: Shall I open a separate issue for that other site? As it's a corporate one I doubt any of you could check there (requires an account of the company), but as DarkReader somehow detects the Confluence behind it (why else should it load confluence-specific resources), it could possibly be cross-checked with any other instance? IMHO loading graphics etc. from multiple domains in different legislations (there e.g. RU) is not the best idea, especially if it slows down loading pages as much as in this case (up to 1h instead of 10s).

**Comment by Myshor** [comment]
You can open bug for site we could check more easily. 😉 But still remember if it won't be able to do fix with sitefixes which can be upgraded even in older Dark Reader and will need some code fixes and new version then you still will need to update to newer version.

**Comment by IzzySoft** [comment]
> You can open bug for site we could check more easily.

Yeah, I'd do that if possible. But I usually don't work with Confluence, only at work – and there that's the only site having these issues.

Btw, where do I see the source of the stylesheet/site-fixes DarkReader generates? In that specific case with the Confluence instance I only found out because someone provided the resulting CSS, but didn't state where it came from. So I picked that one, fixed it up and loaded it via Stylus. Works so-so: better than nothing, but not as good as DarkReader did once the page finally loaded. So if I knew where to find the CSS DarkReader is generating for it, I could check that. My guess there is it's the external resources from different domains which might be blocked/package-dropped by corporate networking stuff causing the huge delays (waiting for timeouts of each resource).

And at work I use the latest version of DarkReader, there were no issues with it there. It's just the fritz.box thingy which you cannot check without a Fritz!Box – and where I run DarkReader 4.9.66.
